cmd/app/options/app.go

```diff
@@ -43,6 +44,10 @@ func (k *KubeOIDCProxyOptions) AddFlags(fs *pflag.FlagSet) *KubeOIDCProxyOptions
 	fs.IntVarP(&k.ReadinessProbePort, "readiness-probe-port", "P", 8080,
 		"Port to expose readiness probe.")
 
+	fs.StringVar(&k.MetricsListenAddress, "metrics-serving-address", "0.0.0.0:80",
+		"Adress to serving metrics on at the /metrics path. An empty address will "+
+			"disable serving metrics.")
```

Comment:
Worth mentioning that this can't conflict with the other addresses here?

Reply:
Also the readiness probe is exposed as a port, but this is a full address. I think that makes sense despite the inconsistency.